frontend/src/components/signup/Signup.component.js

```diff
@@ -158,6 +158,69 @@ const Signup = () => {
             </button>
           </div>
         </div>
+      </div> */}
+
+      <div className="min-h-full min-w-full bg-pixelSepia bg-cover pixel-font">
+        <form
+          className=""
+          onSubmit={handleSubmit}
+          data-test="SignupForm"
+        >
+          <div className="h-screen flex justify-center items-center">
+            <div className="m-7 flex flex-col relative w-2/5">
+              <div className="bg-japanese-brown-2 mt-7 shadow-2xl sm:rounded-xl">
+                <div className="p-5 text-white">
+                  <h1 className="text-8xl md:text-5xl lg:text-7xl">.</h1>
```

Comment:
If this is a design element, maybe it is better not to use h1 tag for a dot?

Reply:
i think ill leave it for now and change it in the future